src/pages/search/ListingCard.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Card, CardBody, CardImg, CardSubtitle, CardText, CardTitle } from 'reactstrap';
+
+import { ListingShort } from 'networking/listings';
+
+const ListingCard = ({
+  homeType,
+  listingPicUrl,
+  pricePerNightUsd,
+  sleepingArrangement,
+  title
+}: ListingShort) => (
+  <Card>
```

Comment:
we can add `tag={Fade}` and import `Fade` from reactstrap to make these cards fade in so they dont appear abruptly